### DISCUSSION TITLE
Many GitHub Issues from a now closed (classic) project have not been added to the (new) project board

### DISCUSSION THREAD

**Discussion by asmith26** [community/community#137634] [Projects and Issues]
### Select Topic Area

Bug / Question

### Feature Area

Projects

### Body

Hi there,

I've noticed that many GitHub Issues from a now closed (classic) project have not been added to the (new) project board following the automatic migration: https://github.blog/changelog/2024-05-23-sunset-notice-projects-classic/

Reading that blogpost, I notice it says:

> Only cards that have been updated in the last year will be migrated.

This is likely the reason; is there any ways can migrate all these tickets as well to the new project board? Currently any such issues show:

![image](https://github.com/user-attachments/assets/188dbc07-4bbf-4cd2-8ccd-e11cfc4fbe58)

Thanks!

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by evi-liu** [comment]
Hi @asmith26, you are correct that only cards that have been updated in the last year were migrated. Could you share the link to your project that was migrated? We can re-run the migration to ensure all of your cards are migrated to your new project board.

**Reply by asmith26** [reply]
Many thanks for your help and info @evi-liu . It's a private project - perhaps there is something I can do myself (e.g. via an API or something)? Thanks again

**Reply by evi-liu** [reply]
Would it be okay if you email me the project link at `evi-liu` `at` `github.com`?

**Reply by asmith26** [reply]
Thanks for the offer @evi-liu . I'm happy to just manually migrate any missing issues.
Thanks again :)
